applications/luci-app-shadowsocks-libev/htdocs/luci-static/resources/view/shadowsocks-libev/servers.js

```diff
@@ -1,21 +1,59 @@
 'use strict';
 'require form';
+'require uci';
```

Comment:
Can you also explicitely require `ui` and use it instead of `L.ui` throughout the code, similar to https://github.com/openwrt/luci/commit/86f492173d1daab8b75e4d4bd70ba06872ea70fa? When testing LuCI I found various race conditions where `ui` might not yet be loaded when the view is processed.